WBooks/Screens/BookDetails/CardDetails/CardDetailsViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  CardDetailsViewController.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 10/05/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class CardDetailsViewController: UIViewController {
+    private let _cardDetailsViewModel: CardDetailsViewModel
+    private let _view: CardDetailsView = CardDetailsView.loadFromNib()!
+    
+    // MARK: - UIViewController
+
+    required public init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        fatalError("init(nibName:bundle:) has not been implemented")
+    }
+    
+    // MARK: - CardDetailsViewController
+
+    init(viewModel: CardDetailsViewModel) {
+        _cardDetailsViewModel = viewModel
+        super.init(nibName: .none, bundle: .none)
+    }
+    
+    override public func loadView() {
+        view = _view
+        _view.setupBookDetails(title: _cardDetailsViewModel.title,
+                               genre: _cardDetailsViewModel.genre,
+                               year: _cardDetailsViewModel.year,
+                               author: _cardDetailsViewModel.author,
+                               status: _cardDetailsViewModel.status,
+                               image: _cardDetailsViewModel.image)
```

Comment:
Normally `loadView` is related to loading a view from a nib. Any further customization goes in `viewDidLoad`.